controller.go

```diff
@@ -125,6 +125,11 @@ func (c Controller) handleHealthcheck(w http.ResponseWriter, r *http.Request) {
 func (c Controller) handleGoodToGo(w http.ResponseWriter, r *http.Request) {
 	categories := parseCategories(r.URL)
 	healthResults, validCategories := c.buildHealthResultFor(categories, useCache(r.URL))
+	// check any any of the categories are disabled
+	enabled := c.catEnabled(validCategories)
```

Comment:
You could return 503 before building health result. This would make it faster for requests with ?cache=false